backend/danswer/main.py

```diff
@@ -90,7 +90,7 @@ def get_application() -> FastAPI:
     )
 
     @application.on_event("startup")
-    async def startup_event() -> None:
+    def startup_event() -> None:
```

Comment:
@yuhongsun96 is there any reason why this was async?

Reply:
not really :P